src/AutoRest.CSharp/Properties/launchSettings.json

```diff
@@ -420,6 +420,10 @@
       "commandName": "Project",
       "commandLineArgs": "--standalone $(SolutionDir)\\test\\TestProjects\\MgmtSubscriptionNameParameter\\Generated"
     },
+    "MixAPIVersion-Cadl": {
```

Comment:
Can we test in the existing test project like [APIVersoin](https://github.com/Azure/autorest.csharp/tree/feature/v3/test/TestProjects/ApiVersion)

Reply:
No. I want to define an special case for this mix scenario, And ApiVersion-Cadl is to test an apiversion as path paramter.